spec.bs

```diff
@@ -38,7 +38,6 @@ spec: ECMAScript; urlPrefix: https://tc39.github.io/ecma262/#
 spec: i18n-glossary; urlPrefix: https://w3c.github.io/i18n-glossary/
```

Comment:
Note that i18n-glossary is in the TR space as https://www.w3.org/TR/i18n-glossary and this probably should be your URL here.